internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -296,6 +297,33 @@ func (e *BootstrapEnvStackConfig) ToEnv(stack *cloudformation.Stack) (*config.En
 	}, nil
 }
 
+func (e *EnvStackConfig) cdnConfig() template.CDNConfig {
+	var (
+		enabled      bool
+		prefixListId *string
+	)
+
+	if e.in.Mft != nil {
```

Comment:
same here

Reply:
👍 can we just return `nil` for now so that it's guaranteed that we don't accidentally include the `cdn` partial to the environment template?